### PR TITLE
WF-63960 : Add more information about adding child items to a group in navigation pane

### PR DESCRIPTION
**Pull request by Karthick-Arjunan**
### Description
Add more information about adding child items in to group bar.

### CODE REVIEW

File: WindowsForms/Navigation-Pane/GettingStarted.md [hunk 1]
```diff
@@ -67,61 +67,106 @@ Me.Controls.Add(groupBar1)
...
-            Me.groupBarItem3,
-            Me.groupBarItem4,
-            Me.groupBarItem5})
+            Me.groupBarItem2})
 {% endhighlight %}
 {% endtabs %}
 
 ![wf group bar control added in designer](Getting-Started_images/wf-group-bar-control.png)
 
+## Add sub child items in group bar
```
Comment (Manivannan-E) [inline WindowsForms/Navigation-Pane/GettingStarted.md:109]: @Karthick-Arjunan Add child items to the group bar items.

File: WindowsForms/Navigation-Pane/GettingStarted.md [hunk 2]
```diff
@@ -73,54 +73,169 @@ You can add the group bar items inside the Groupbar control using the [GroupBarI
...
+            Me.groupBarItem0,
+            Me.groupBarItem1,
+            Me.groupBarItem2})
+{% endhighlight %}
+{% endtabs %}
+
+![wf group bar control](Getting-Started_images/wf-group-bar-control.png)
+
+## Add child items to the group bar items.
+You can add the child items to the GroupbarItems. The following code snippet illustrates how to add the child items to the GroupbarItems.
```
Comment (Manivannan-E) [inline WindowsForms/Navigation-Pane/GettingStarted.md:110]: @Karthick-Arjunan Change the content as we discussed. There is no proper explanation.

File: WindowsForms/Navigation-Pane/GettingStarted.md [hunk 3]
```diff
@@ -73,54 +73,169 @@ You can add the group bar items inside the Groupbar control using the [GroupBarI
...
-            Me.groupBarItem3,
-            Me.groupBarItem4,
-            Me.groupBarItem5})
+            Me.groupBarItem2})
+
 {% endhighlight %}
 {% endtabs %}
 
-![wf group bar control added in designer](Getting-Started_images/wf-group-bar-control.png)
+![wf group bar control in default mode](Getting-Started_images/wf-group-bar-control-default-mode.png)
```
Comment (Manivannan-E) [inline WindowsForms/Navigation-Pane/GettingStarted.md:238]: @Karthick-Arjunan WinForms GroupBar control with group bar items.

File: WindowsForms/Navigation-Pane/GettingStarted.md (hunk 1 above)
Comment (Karthick-Arjunan) [reply WindowsForms/Navigation-Pane/GettingStarted.md]: Changed the content.

File: WindowsForms/Navigation-Pane/GettingStarted.md (hunk 3 above)
Comment (Karthick-Arjunan) [reply WindowsForms/Navigation-Pane/GettingStarted.md]: Changed the image name.

File: WindowsForms/Navigation-Pane/GettingStarted.md (hunk 2 above)
Comment (Karthick-Arjunan) [reply WindowsForms/Navigation-Pane/GettingStarted.md]: Changed the content.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**